Tests/Weaver/AssemblyToProcess/ClassWithStaticProperty.cs

```diff
@@ -26,12 +27,13 @@ public class Dog : RealmObject
     {
         public string Name { get; set; }
 
-        private static Dog myDog;
+        private static readonly Dog myDog = new Dog();
 
         public static Dog MyOnlyDog
         {
             get
             {
+                Console.WriteLine("");
```

Comment:
Why was this needed?

Reply:
Apf... this was a silly way to silence a VS message that this should be an auto property. I'll change it to be more meaningful.